src/Tickets/Blocks/Tickets/app/editor/action-dashboard/container.js

```diff
@@ -23,10 +23,12 @@ const mapStateToProps = (state, ownProps) => {
 	const provider = selectors.getTicketsProvider(state);
 	const page = constants.TICKET_ORDERS_PAGE_SLUG[provider];
 	const isRecurring = hasRecurrenceRules(state);
+	const selectedBlock = select( 'core/editor' ).getSelectedBlock();
 
 	let mappedProps = {
 		hasCreatedTickets: selectors.hasCreatedTickets(state),
 		hasOrdersPage: Boolean(page),
+		isBlockSelected: selectedBlock?.name === 'tribe/tickets',
```

Comment:
Why not use `ownProps.isSelected`?

Reply:
ownProps.isSelected didn't work, if I select the block that doesn't appear as `true`